scripts/run_docker_android_base.sh

```diff
@@ -1,3 +1,4 @@
+#!/bin/sh
 sudo docker run -ti --privileged \
     -v /dev/bus/usb:/dev/bus/usb \
     android-base \
```

Comment:
Change android - base to add samtstern prefix.   This is my mistake but we may as well fix it here.